addons/mail/static/src/discuss/core/common/action_panel.scss

```diff
@@ -1,3 +1,7 @@
+.o-mail-ActionPanel {
+    max-width: 45vw;
+}
+
```

Comment:
This rule doesn't work for the chatter : 
![image](https://github.com/user-attachments/assets/457db5ad-2848-4ea5-8c11-c32d8943fae7)